.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,15 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
```

Comment:
When contributors are at this step (sending a PR), they may modify the docs in the wrong way already. I suggest to make them see the tips as early as possible. What I expected in #1121 is adding some tips in each README of the sigs. We can add a field `tip` in `sigs.yaml` and auto generate it to all the READMEs. Contributors can see what they should do when they want to make their change on a README.

Reply:
We already do this with comments at the top of every README (see [here](https://raw.githubusercontent.com/kubernetes/community/master/sig-api-machinery/README.md)), so folks will know by the time they come to edit the file. 

Do you mean make these tips visible on each SIG README too? I'm wondering how that adds more value than just embedding stuff in the comment section, since they'll see it.

Reply:
Yes I meant this. But since the tips are already in the comments (sorry I didn't notice that :), it's not necessary to make them visible.